lotto/src/main/kotlin/domain/LottoStrategy.kt

```diff
@@ -0,0 +1,11 @@
+package domain
+
+fun interface LottoStrategy {
+    fun generateNumbers(): List<Int>
+}
+
+class AutoLottoStrategy : LottoStrategy {
+    override fun generateNumbers(): List<Int> {
+        return (1..45).toList().shuffled().subList(0, 6)
+    }
```

Comment:
```suggestion
    override fun generateNumbers(): List<Int> {
        return (1..45).toList().shuffled().take(6)
    }
```

`take()`를 사용해볼 수도 있을 것 같아요. `take` 는 코틀린에서 제공하는 확장함수 입니다!

Reply:
와 뭔 이런게 다 있대요...

Reply:
없는 게 없는 코틀린.....